db-client-java/src/main/java/com/eventstore/dbclient/EventStoreDBPersistentSubscriptionsClient.java

```diff
@@ -164,4 +176,243 @@ public CompletableFuture deleteToAll(String group, DeletePersistentSubscriptionO
+
+                return Optional.of(parseSubscriptionInfo(node));
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            } finally {
+                http.disconnect();
+            }
+        });
+    }
+
+    public CompletableFuture<Optional<PersistentSubscriptionInfo>> getInfo(String stream, String groupName) {
+        return getInfo(stream, groupName, GetPersistentSubscriptionInfoOptions.get());
+    }
+
+    public CompletableFuture replayParkedMessages(String stream, String groupName, ReplayParkedMessagesOptions options) {
+        String query;
+
+        if (options.getStopAt() != null) {
+            query = "?stopAt=" + options.getStopAt().getSeconds();
+        } else {
+            query = "";
+        }
+
+        return getHttpURLConnection(options, "/subscriptions/" + stream + "/" + groupName + "/replayParked" + query).thenApply(http -> {
+            try {
+                http.setDoOutput(true);
+                http.setRequestMethod("POST");
+                http.setFixedLengthStreamingMode(0);
+
+                throwOnError(http.getResponseCode());
+                return null;
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            } finally {
+                http.disconnect();
+            }
+        });
+    }
+
+    public CompletableFuture replayParkedMessages(String stream, String groupName) {
+        return replayParkedMessages(stream, groupName, ReplayParkedMessagesOptions.get());
+    }
+
+    public <A> CompletableFuture<HttpURLConnection> getHttpURLConnection(OptionsBase<A> options, String path) {
+        return this.client.getCurrentEndpoint().thenApply(edp -> {
+            try {
+                HttpURLConnection conn = (HttpURLConnection) edp.getURL(this.client.settings.isTls(), path).openConnection();
+                conn.setRequestProperty("Accept", "application/json");
+                String creds = options.getUserCredentials();
+
+                if (creds == null && this.client.settings.getDefaultCredentials() != null) {
+                    creds = this.client.settings.getDefaultCredentials().toUserCredentials().basicAuthHeader();
+                }
+
+                if (creds != null) {
+                    conn.setRequestProperty("Authorization", creds);
+                }
+
+                return conn;
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        });
+    }
+
+    private static String readContent(HttpURLConnection conn) throws IOException {
+        try (BufferedReader in = new BufferedReader(new InputStreamReader(conn.getInputStream()))) {
+            String line;
+            StringBuilder content = new StringBuilder();
+            while ((line = in.readLine()) != null) {
+                content.append(line);
+            }
+
+            return content.toString();
+        } catch (IOException e) {
+            throw e;

```

Comment:
**CONCERN:** I might not be a Java specialist here, but isn't this rethrow redundant if we're just rethrowing IOException, that we already had defined by `throws IOException` in the method signature?

Reply:
You are right.